helical/models/hyena_dna/standalone_hyenadna.py

```diff
@@ -881,9 +881,7 @@ def __init__(self, d_model: int, n_layer: int, d_inner: int, vocab_size: int,
 
         self.use_head = use_head
 
-        # check if layer (config) has d_model (HF code differs from main Safari code)
-        if 'd_model' not in layer:
-            layer['d_model'] = d_model
+        layer['d_model'] = d_model
```

Comment:
Because `d_model` is not defined in the layer of `HyenaDNAConfig` we assign it always from the `d_model` variable. (which we infer in the HyenaDNAConfig).